src/index.ts

```diff
@@ -269,6 +269,17 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "The name, without the architecture, of a gitlab hosted runner to emulate. See here: https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#machine-types-available-for-linux---x86-64",
             choices: GitlabRunnerPresetValues,
         })
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        // NOTE: not redundant, this is added for the `--help` message
+        .option("no-color", {
+            requiresArg: false,
+            description: "Disable all color",
+        })
```

Comment:
This is redundant. There is a description in the top of help showing all commands can be negated with `--no`
```suggestion
```

Reply:
ahh i see your point, i just noticed the description haha